bammetrics/src/main/resources/nl/lumc/sasc/biopet/pipelines/bammetrics/rnaHistogram.ssp

```diff
@@ -62,27 +61,25 @@
         #end
     </tr></thead>
     <tbody>
-        #for (sample <- samples.toList.sorted)
+        #for (sample <- samples.sortBy(_.name))
```

Comment:
This section gets repeated in several places. To prevent code duplications, perhaps it's better to place a function for this in utils.